### PR TITLE
add 3 array methods in javascript cheatsheet

### PR DESCRIPTION
**Pull request by MaxR522**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/JavaScript-CheatSheet.md [hunk 1]
```diff
@@ -90,7 +90,9 @@ var names= ["Raj", "Ram", "Sham"];
 | splice()      | Adds elements in a specified way and position                           |
 | toString()    | Converts elements to strings                                            |
 | unshift()     | Adds a new element to the beginning                                     |
-| valueOf()     | Returns the primitive value of the specified object                     |
```
Comment (crescentpartha) [inline CheatSheets/JavaScript-CheatSheet.md:93]: You deleted the `valueOf()` column. Please, add this line or tell me why you deleted it.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify your code or answer my question.

**Comment by MaxR522** [comment]
> Please, modify your code or answer my question.

Sorry!!
My bad, I will modify it

**Comment by MaxR522** [comment]
I fixed it in this commit [cfce718](https://github.com/crescentpartha/CheatSheets-for-Developers/pull/166/commits/cfce71892e44cebe77a510ed42b2f6a90ae8a9f1)

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**